edx_proctoring/views.py

```diff
@@ -950,12 +950,19 @@ def get(self, request, course_id, exam_id=None):
         exam = None
         attempt_id = None
         ext_exam_id = None
+        show_configuration_dashboard = ''
+
         if exam_id:
             exam = get_exam_by_id(exam_id)
             # the exam_id in the url is our database id (for ease of lookups)
             # but the backend needs its external id for the instructor dashboard
             ext_exam_id = exam['external_id']
             attempt_id = request.GET.get('attempt', None)
+
+            # only show the configuration dashboard if an exam_id is passed in
+            show_configuration_dashboard = request.GET.get('config', '').lower()
+            show_configuration_dashboard = True if show_configuration_dashboard == 'true' else False
```

Comment:
this is the same as `show_configuration_dashboard = request.GET('config', '').lower() == 'true'`